source/docs/03-metadata.xml

```diff
@@ -900,15 +900,16 @@
                      <specDesc key="work"/>
                   </specList>
                </p>
-               <p>All the components of <gi scheme="MEI">work</gi> are optional, but they must occur in the following order:</p>
+               <p>All the components of <gi scheme="MEI">work</gi> are optional. They may occur in any order.</p>
                <list rend="numbered">
```

Comment:
```suggestion
               <list rend="bulleted">
```

Reply:
Sorry, just recognized this one.

At the moment, there is no difference in the rendering between numbered or bulleted lists, but maybe at some point there will be. Since the components here are not strictly ordered anymore, we should also "relax" the listing.

Reply:
There is, but it must "ordered", not "numbered".

Reply:
After investigation through the mei guidelines there is `list/@rend` with the following values: `simple` (one time), `numbered` and `bulleted`. What I like to see here is `bulleted`.